mobile/mysterium/wireguard_connection_setup.go

```diff
@@ -28,10 +28,10 @@ import (
 	"github.com/mysteriumnetwork/node/core/connection"
 	"github.com/mysteriumnetwork/node/services/wireguard"
 	"github.com/mysteriumnetwork/node/services/wireguard/key"
-	"github.com/mysteriumnetwork/wireguard-go/device"
```

Comment:
From this, it was clear, that we are using our version of wireguard-go. And now it looks like we are using original packages which is confusing.

Reply:
The goal is to integrate sooner into upstream. It is only logical that we keep original packages.